base/dcp_feed.go

```diff
@@ -517,6 +518,16 @@ func (r *DCPLoggingReceiver) initFeed(feedType uint64) error {
 	return r.rec.initFeed(feedType)
 }
 
+// NoPasswordAuthHandler is used for client cert-based auth
+type NoPasswordAuthHandler struct {
+	handler AuthHandler
+}
+
+func (nph NoPasswordAuthHandler) GetCredentials() (string, string, string) {
```

Comment:
Could the params be named?  (I realized the existing code has the same issue, but this might be a good time to improve)

Reply:
Agreed - fixed.